internal/cmd/local/localerr/localerr.go

```diff
@@ -1,23 +1,67 @@
 package localerr
 
-import "errors"
+var _ error = (*LocalError)(nil)
+
+// LocalError adds a user-friendly help message to specific errors.
+type LocalError struct {
```

Comment:
I wonder if it would be better as an interface, which we could implement on any custom error types we have. hmmm.

Reply:
I debated that as well, but didn't see a need for it yet. It would be easy enough to change if we decide to do that later.